ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -928,6 +1235,80 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'DSR Dive From Grace Dive Position',
+      type: 'GainsEffect',
+      // AC3 = High Jump Target
+      // AC4 = Spineshatter Dive Target
+      // AC5 = Elusive Jump Target
+      // Defaults:
+      //   Spineshatter West, Elusive East, All Face East
+      //   High Jump North if solo, no assignment if all circle
+      //   2s Southeast/Southwest, no assignment if circle
+      //   Assumes North Party Stack
+      //
+      // Spineshatter and Elusive elusive come together
+      // Circle never paired with single Spine/Elusive
+      netRegex: NetRegexes.gainsEffect({ effectId: ['AC3', 'AC4', 'AC5'] }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 5,
+      alertText: (data, matches, output) => {
+        const num = data.diveFromGraceNum[data.me];
+        if (!num) {
+          console.error(`DFG Dive Position: missing number: ${JSON.stringify(data.diveFromGraceNum)}`);
+          return;
+        }
+
+        // Output no direction when all circles
+        if (!data.diveFromGraceHasArrow[num]) {
+          if (num === 2)
+            return output.diveCircles2!();
+
+          // Can predict where num3s will go based on Darkdragon Dive targeting
+          return output.diveCircles!();
+        }
+
+        // Output West or East
+        if (num === 1 || num === 3) {
+          if (matches.effectId === 'AC3')
+            return output.diveCircle!();
+          if (matches.effectId === 'AC4')
+            return output.diveSpineshatter!();
+          return output.diveElusive!();
+        }
+
+        // By the time 2s turn, they will be stacked, facing boss,
+        // so calls are relative to the boss
+        if (num === 2) {
+          if (matches.effectId === 'AC4')
+            return output.diveSpineshatter2!();
+          return output.diveElusive2!();
+        }
+      },
+      outputStrings: {
+        diveCircles: {
+          en: 'Cardinal Dive',
+        },
+        diveCircle: {
+          en: 'South Dive',
+        },
+        diveCircles2: {
+          en: 'Intercard Dive',
+        },
+        diveSpineshatter: {
+          en: 'West Dive, Face Boss',
+        },
+        diveElusive: {
+          en: 'East Dive, Face Out',
+        },
+        diveSpineshatter2: {
+          en: 'Backright Dive, Face East',
```

Comment:
So, I have seen different people do different strats here and face west.  Could this be "up arrow dive" and folks can rewrite it if they want to have something more specific?

I think that would then make "circle dive" a more natural call for the 2s and 1s where it's unclear.  I had to read "intercard dive" a few times to figure out what that meat.